sdk/Lusid.Sdk.Tests/Utilities/TestDataUtilities.cs

```diff
@@ -719,10 +721,14 @@ internal static void CheckNonZeroPvBeforeMaturityAndZeroAfter(ListAggregationRes
                 {
                     Assert.That(pv, Is.Not.EqualTo(0).Within(1e-12));
                 }
-                else
+                if (date > maturityDate)
                 {
                     Assert.That(pv, Is.EqualTo(0).Within(1e-12));
                 }
+                if (date == maturityDate)
+                { // ANA-1292: Currently, we do not perform an assertion if the date is exactly the maturity date, as it's not clear what the PV should be.
```

Comment:
I don't prefer adding ANA-XXXX here on GH as it is public.

Reply:
removed